pep-0656.rst

```diff
@@ -54,6 +62,48 @@ Tags using the new scheme will take the form::
 
     musllinux_${MUSLMAJOR}_${MUSLMINOR}_${ARCH}
 
+Reading the musl version
+------------------------
+
+The musl version values can be obtains by executing the musl libc
+shared library the Python interpreter is currently running on, and
+parsing the output::
+
+    import re
+    import subprocess
+
+    def get_musl_major_minor(so: str) -> tuple[int, int] | None:
+        """Detect musl runtime version.
+
+        Returns a two-tuple ``(major, minor)`` that indicates musl
+        library's version, or ``None`` if the given lib .so does not
```

Comment:
Ah my bad, I made a typo in my suggestion :facepalm: 
```suggestion
        library's version, or ``None`` if the given libc .so does not
```

Reply:
I amended the last commit 🙂

Reply:
I’ve amended the last commit 🙂